src/WebJobs.Extensions.DocumentDB/Bindings/DocumentDBEnumerableValueBinder.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
+{
+    internal class DocumentDBEnumerableValueBinder<T> : IValueBinder
+    {
+        private DocumentDBContext _context;
+
+        public DocumentDBEnumerableValueBinder(DocumentDBContext context)
+        {
+            _context = context;
+        }
+
+        public Type Type
```

Comment:
could use an expression bodied member to get a little more compact

`public Type Type => typeof(IEnumerable<T>)`